mod/01_rating/set/ltype_rtype/metric/wikirate_company.rb

```diff
@@ -5,37 +5,42 @@ def latest_value_year
 end
 
 def latest_value_card
-  if latest_value_year != 0
-     Card.fetch "#{name}+#{latest_value_year}"
-  end
+  return if !(lvy = latest_value_year) || lvy == 0
+  Card.fetch "#{name}+#{latest_value_year}"
 end
 
-expired_cached_count_cards :set=>Card::Set::Type::MetricValue do |changed_card|
-   changed_card.left
+expired_cached_count_cards set: Card::Set::Type::MetricValue do |changed_card|
```

Comment:
Pass `&:left` as an argument to `expired_cached_count_cards` instead of a block.